typespec/Applications.Dapr/common.tsp

```diff
@@ -33,4 +33,7 @@ model DaprResourceProperties {
 
   @doc("Dapr component version")
   version?: string;
+
```

Comment:
Sure! I'll also add types to the Metadata field in the Dapr components data model as well, if that’s okay with you. This might help prevent some "casting to any" errors.